src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializer.Write.ByteArray.cs

```diff
@@ -24,7 +24,7 @@ public static byte[] SerializeToUtf8Bytes<TValue>(TValue value, JsonSerializerOp
         /// <param name="value">The value to convert.</param>
         /// <param name="inputType">The type of the <paramref name="value"/> to convert.</param>
         /// <param name="options">Options to control the conversion behavior.</param>
-        public static byte[] SerializeToUtf8Bytes(object value, Type inputType, JsonSerializerOptions options = null)
+        public static byte[] SerializeToUtf8Bytes(object? value, Type? inputType, JsonSerializerOptions? options = null)
```

Comment:
Type should not be nullable. We should consider fixing the validation to throw in that case.

```suggestion
        public static byte[] SerializeToUtf8Bytes(object? value, Type inputType, JsonSerializerOptions? options = null)
```